addons/binding/org.openhab.binding.gpstracker/README.md

```diff
@@ -152,6 +174,127 @@ sitemap gpstracker label="GPSTracker Binding" {
     Text item=atHomeEX
     Text item=lastSeenEX
     Text item=batteryEX
+    Text item=accurayEX
     Mapview item=locationEX height=4
 }
 ```
+
+## Debug
+
+As the setup is not that simple here are some hints for debugging.
+In order to see the debug information in log enable TRACE for the binding in the console:
```

Comment:
I am not sure if we should add this information to the binding docs. Rather link to https://www.openhab.org/docs/administration/logging.html

Reply:
Not clear what to remove: only this first paragraph or the entire Debug section?

Reply:
The section in general is okay. Please just replace the console command with the link the docs in my first comment.

Reply:
Done. I hope it's ok.